package com.github.member.exception

/**
 * @author Andochiwa
 * @version 1.0
 * @since 12-04-20:09
 */
class UsernameExistException : RuntimeException("用户名已存在")
